src/frontend/src/common/components/library/data_subview/index.tsx

```diff
@@ -259,7 +266,7 @@ const DataSubview: FunctionComponent<Props> = ({
           <Divider />
           <IconButton
             onClick={handleCreateTreeClickOpen}
-            disabled={isDownloadDisabled}
+            disabled={isCreateTreeDisabled}
```

Comment:
it seems like making this change causes `isDownloadDisabled` to not be read/used anywhere. is that intentional? If so, you may want to remove that state variable.

Reply:
isDownloadDisabled is used in the following IconButton component for the DownloadModal

Reply:
I think I might be confused, but I don't see it there. Maybe we can pair real quick and sync on it ...

Reply:
it's on line 277 in this file?

Reply:
LOL idk wtf happened there. I searched in my editor and it wasnt coming up

Reply:
haha no worries at all!!